### PR TITLE
Provide a script to update version of Ankaios

### PR DESCRIPTION
**Pull request by windsource**
Issue-Id: #337

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [ ] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

File: tools/update_version.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+done
+
+# Check if VERSION is set
+if [[ -z "$version" ]]; then
+    echo "Error: VERSION is a mandatory argument."
+    usage
+fi
+
+# Extract all packages from the workspace file
+packages=$(awk '/members *= *\[/{flag=1; next} /\]/{flag=0} flag {gsub(/[" ,]/, ""); print}' "$workspace_config")
```
Comment (inf17101) [inline tools/update_version.sh:62]: To be more robust against newlines between package names, I suggest the following small change:
```suggestion
packages=$(awk '/members *= *\[/{flag=1; next} /\]/{flag=0} flag && NF {gsub(/[" ,]/, ""); print}' "$workspace_config")
```

**Review by inf17101 — COMMENTED** [review]
Review finished. Two small findings.

A hint inside the prepare release documentation that this script shall be executed would be useful: [Prepare Release](https://eclipse-ankaios.github.io/ankaios/main/development/ci-cd-release/#preparing-a-release)

File: tools/update_version.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Elektrobit Automotive GmbH
```
[the remark this answers is not in the document]
Comment (windsource) [reply tools/update_version.sh:3]: OK.

File: tools/update_version.sh (hunk 1 above)
Comment (windsource) [reply tools/update_version.sh:62]: I have tested the existing script and it already works with empty lines between the members. The awk statement will then just output an empty item which will be ignored by the line `for pkg in $packages; do`.
For that reason I think we not need to include this change.

Comment (inf17101) [reply tools/update_version.sh:62]: Ok, I think we can keep it.

**Review by inf17101 — APPROVED** [review]
LTGM👍

**Pull request merged**